kitchen.yml

```diff
@@ -1,9 +1,9 @@
 ---
 provisioner:
   deprecations_as_errors: true
+  multiple_converge: 2
```

Comment:
This isn't very useful without `enforce_idempotency: true`. Now I"m not sure how good this cookbook is with that. If it's a lot of work to fix idempotency, then I'd say skip it for this PR. But I'd try it locally on a few suites and see how far the rabbit hole is and if it's too big, then skip it and remove this line.

Reply:
I checked it and youre right, a lot fail with idempotency checks so I removed the multiple_converge

Reply:
This actually is nesecary for some tests that check on things that should exist after splunk is restarted. Should I leave in multiple converge or try to manually restart it during tests?

Reply:
Ok, let's leave it but leave a comment for why.